tests/testthat/test-simplify.R

```diff
@@ -304,3 +304,13 @@ test_that("ms_simplify works with various column types", {
   xsf$check_me <- raw(nr)
   expect_warning(simp_xsf <- ms_simplify(xsf), "NAs introduced by coercion")
 })
+
+
+# units -------------------------------------------------------------------
+
+test_that("ms_simplify works with sf objects containing units", {
+  multipoly_sf$area = sf::st_area(multipoly_sf)
+  multipoly_sf_simple = ms_simplify(multipoly_sf)
```

Comment:
can you also wrap this in `expect_warning()`
```suggestion
  expect_warning(multipoly_sf_simple <- ms_simplify(multipoly_sf), "units")
```